src/test/java/ay2021s1_cs2103_w16_3/finesse/testutil/FinanceTrackerBuilder.java

```diff
@@ -0,0 +1,34 @@
+package ay2021s1_cs2103_w16_3.finesse.testutil;
+
+import ay2021s1_cs2103_w16_3.finesse.model.FinanceTracker;
+import ay2021s1_cs2103_w16_3.finesse.model.transaction.Transaction;
+
+/**
+ * A utility class to help with building FinanceTracker objects.
+ * Example usage: <br>
+ *     {@code FinanceTracker ab = new FinanceTrackerBuilder().withTransaction("John", "Doe").build();}
```

Comment:
Perhaps the variable name should be `ft` instead of `ab`?

Also I don't see any method signature for `withTransaction(String, String)`. Is this a problem inherited from the original AB3?

Reply:
The example usage in the original AB3 was already erroneous, so I will be removing this example usage.